examples/simple_muscles.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import sympy as sm
+import sympy.physics.mechanics as me
+from sympy.physics.mechanics.pathway import LinearPathway
+
+from biomechanics import (FirstOrderActivationDeGroote2016,
+                          MusculotendonDeGroote2016)
+from biomechanics.plot import plot_config, plot_traj
+
+q, u = me.dynamicsymbols('q, u')
+m, g = sm.symbols('m, g')
+F_M_max, l_M_opt, l_T_slack = sm.symbols('F_M_max, l_M_opt, l_T_slack')
+v_M_max, alpha_opt, beta = sm.symbols('v_M_max, alpha_opt, beta')
+
+N = me.ReferenceFrame('N')
+O, P = sm.symbols('O, P', cls=me.Point)
+
+P.set_pos(O, q*N.x)
+O.set_vel(N, 0)
+P.set_vel(N, u*N.x)
+
+gravity = me.Force(P, m*g*N.x)
+
+muscle_pathway = LinearPathway(O, P)
+muscle_activation = FirstOrderActivationDeGroote2016('big_dog').with_default_constants('muscle')
+muscle = MusculotendonDeGroote2016(
+    'muscle',
+    muscle_pathway,
+    activation_dynamics=muscle_activation,
+    tendon_slack_length=l_T_slack,
+    peak_isometric_force=F_M_max,
+    optimal_fiber_length=l_M_opt,
+    maximal_fiber_velocity=v_M_max,
+    optimal_pennation_angle=alpha_opt,
+    fiber_damping_coefficient=beta,
+)
```

Comment:
Add the tendon slack length and optimal fiber length and this should match the initial pathway length.